editoast/src/redis_utils.rs

```diff
@@ -46,6 +49,30 @@ impl ConnectionLike for RedisConnection {
     }
 }
 
+impl RedisConnection {
+    pub async fn json_get<T: DeserializeOwned, K: ToRedisArgs + Send + Sync>(
+        &mut self,
+        key: K,
+    ) -> Result<Option<T>> {
+        let value: Option<String> = self.get(key).await?;
+        match value {
+            Some(v) => Ok(serde_json::from_str(&v)?),
```

Comment:
We should transform the `SerdeError` into a `Redis` custom error (we can type it as `Serialize`). This will throw a 500 instead of a 400.